worlds/sc2wol/Options.py

```diff
@@ -42,7 +42,8 @@ class FinalMap(Choice):
     Vanilla mission order always ends with All in mission!
 
     Warning: Using All-in with a short mission order (7 or fewer missions) is not recommended,
-        as there might not be enough locations to place all the required items.
+        as there might not be enough locations to place all the required items,
+        any excess required items will be placed into starting inventory!
```

Comment:
Missing an article, maybe "items will be placed into [the player's] starting inventory"